### DISCUSSION TITLE
نسخه جدید IOS 😒

### DISCUSSION THREAD

**Discussion by persian85** [getlantern/lantern-forum-fa#5] [General]
چند روز پیش تو کانال اعلام کردن نسخه جدید پلتفرم IOS منتشر شده ولی هیچگونه نسخه جدیدی در اپ استور نیست! سرکاریه یا معنی نسخه جدید عوض شده خبر نداریم؟

**Comment by f3ngk** [comment]
فانوس تحریک کننده
